demo_video_command _list

```diff
@@ -0,0 +1,79 @@
+/start
+
+/klip geniusz
+
+/dostosuj 10 10
+
+/szukaj geniusz
+
+/wybierz 2
+
+/lista
+
+/wybierz 8
+
+/kompiluj 1-20
+
+/wytnij S07E06 36:47.50 36:49.00
+
+/zapisz akuku
+
+/mojeklipy
+
+------
+1️⃣ | 📺 S04E07 | 🕒 17.89s
+   👉 szuja
+
+2️⃣ | 📺 S08E03 | 🕒 13.45s
+   👉 warto_doświadczenie_mieć
+
+3️⃣ | 📺 S07E12 | 🕒 15.65s
+   👉 geniusz
+
+4️⃣ | 📺 S01E12 | 🕒 15.74s
+   👉 megalomania
+
+5️⃣ | 📺 S08E11 | 🕒 31.04s
+   👉 17%
+
+6️⃣ | 📺 S09E13 | 🕒 18.92s
+   👉 tabun_dzikich_koni
+-------
+/wyślij akuku
+
+/usuńklip akuku
+
+/mojeklipy
+
+/wyślij 17%
+
+/mojeklipy
+
+/polaczklipy szuja geniusz
+
+/zapisz szuja-geniusz
+
+/mojeklipy
+
+/report znalazłam taki i taki błąd admin weź to załataj
+
+/subskrypcja
+
+/odcinki 5
+
+/odcinki 10
+
+/odcinki 11 - to sobie obczajcie sami :)
+
+/transkrypcja Solejukowski
+
+/start skroty
+
+/klip jola jadziem tabun dzikich koni
+
+/dostosuj 5 18
+
+
```

Comment:
to w ogole ma tu byc? XD

Reply:
znaczy to jest taki scenariusz do dema tak go dałem żeby było w jednym miejscu ale mogę wyjebać

Reply:
w sumie moze zostac ig, tylko no zmien nazwe i moze wrzuc w jakis folder zeby bylo wiadomo ze to helper util a nie cos waznego